Project.toml

```diff
@@ -1,10 +1,16 @@
 name = "Match"
 uuid = "7eb4fadd-790c-5f42-8a69-bfa0b872bfbf"
-version = "1.2.0"
-author = "Kevin Squire <kevin.squire@gmail.com>"
+version = "2.0.0"
+authors = ["Neal Gafter <neal.gafter@relational.ai>", "Kevin Squire <kevin.squire@gmail.com>"]
+
+[deps]
+MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 
 [compat]
-julia = "0.7, 1"
+MacroTools = "0.4, 0.5"
+OrderedCollections = "1"
+julia = "1.4"
```

Comment:
Your PR description says "requires julia 1.5", but your Project.toml (and CI) support 1.4.

Update whichever is wrong?